views/js/runner/plugins/controls/timer/timer.js

```diff
@@ -194,6 +194,8 @@ define([
                     currentTimers[type]
                         .init()
                         .render(self.$element);
+
+                    self.trigger('addtimer', type, currentTimers[type]);
```

Comment:
Could you document the event using the : 

```js
/**
  * @event timerPlugin#addtimer
  * @param {String} type .....
```
This helps us to maintain the documentation of the events. Events are like the API as they are the way to communicate.